src/Route.php

```diff
@@ -133,9 +94,9 @@ public function __construct(string $method, string $path)
      * Add path
      *
      * @param string $path
-     * @return self
+     * @return static
      */
-    public function path(string $path): self
+    public function path(string $path): static
```

Comment:
Here as well 🤔  We are not using this function in a static context, so it makes more sense to use `self` as the return type.